spec.md

```diff
@@ -228,10 +228,40 @@ how platforms might expose these values to their users.
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, service instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether service instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schema-object) | object | Schema definitions for service instances and bindings for the plan. |
```

Comment:
all of these `object`s, should they be `JSON object` to be consistent with what I above? .e.g `metadata`

Reply:
yes it should be - thanks!

Reply:
Wait - I'm not actually sure it should be. Elsewhere in the spec when the object definition is defined in the spec, we just use `object`, and use `JSON object` where the structure is not defined.

Reply:
hmm I think you're right.  That distinction was totally lost on me and I wonder whether its one worth keeping - after all, `object` is always a `JSON object`, right?

Reply:
But that can be a different PR, if we want to change it

Reply:
Yeah, I don't think there are any cases now in the spec where object != JSON object, but we can handle that elsewhere :)